webapp/src/actions.js

```diff
@@ -174,6 +175,26 @@ export const list = (reminder = false, listName = 'my') => async (dispatch, getS
     return {data};
 };
 
+export const count = () => async (dispatch, getState) => {
```

Comment:
I'm thinking this can be a bit more specific. The rest of the functions in this file are mostly one-word verbs, so it may make sense to be consistent with that, but I'm thinking `count` is a bit too vague

```suggestion
export const fetchIssueCounts = () => async (dispatch, getState) => {
```